src/core/dbus-unit.c

```diff
@@ -2080,6 +2081,24 @@ static int bus_unit_set_live_property(
         /* Handles setting properties both "live" (i.e. at any time during runtime), and during creation (for
          * transient units that are being created). */
 
+        if (streq(name, "DebugWait")) {
+                bool d;
+
+                r = sd_bus_message_read(message, "b", &d);
```

Comment:
boolean values sd_bus_message_read() reads must be of type "int", not "bool".

(we strive for a certain level of pre-C99 compat with the public sd-*.h apis, such as sd-bus, hence do not use C99 bool in the public facing interfaces, only in internal code